packages/datadog-plugin-fastify/src/find-my-way.js

```diff
@@ -10,6 +10,7 @@ function createWrapOn () {
       const wrapper = function (req) {
         web.patch(req)
         web.enterRoute(req, path)
+        req._datadog._dd_path_added_by_find_my_way = true
```

Comment:
Can the name be more generic? The logic assumes that some router already handled routing, but it's not necessarily find-my-way.

Reply:
Called it `_dd_route_entered`.